shared/pkg/scanner/grype/common.go

```diff
@@ -206,7 +210,7 @@ func getDescription(match grype_models.Match) string {
 	return match.Vulnerability.Description
 }
 
-func getLayerIDAndPath(coordinates []syft_source.Coordinates) (layerID, path string) {
+func getLayerIDAndPath(coordinates []syft_source.Coordinates) (string, string) {
```

Comment:
I think we can keep param names in the return so it will be clear what is the first string and what is the second one